ap_ruby_sdk/lib/ap_ruby_sdk/transaction.rb

```diff
@@ -7,7 +7,6 @@ class Transaction < ApiResource
     attr_accessor :customer,
                   :customerId,
                   :payment,
-                  :token,
```

Comment:
I thought those were 2 different tokens. I guess we wait for @AlternativePayments  to answer.